packages/create-pantheon-decoupled-kit/src/templates/next-drupal-app/app/[lang]/layout.tsx

```diff
@@ -0,0 +1,73 @@
+import '@pantheon-systems/nextjs-kit/style.css';
+
+import { Footer } from '@pantheon-systems/nextjs-kit/dist/components/footer';
+import { Header } from '@pantheon-systems/nextjs-kit/dist/components/header';
+import styles from './layout.module.css';
+import {
+	getCurrentLocaleStore,
+	globalDrupalStateStores,
+} from '../../lib/stores';
```

Comment:
The default tsconfig sets up a path alias of `@` for the root dir, and after adding the `src` dir you can import like this:
```suggestion
} from '@/src/lib/stores';
```